docs/site/Transactions.md

```diff
@@ -23,10 +23,10 @@ is attached to one of the following connectors:
 - [PostgreSQL connector](PostgreSQL-connector.html)
 - [SQL Server connector](SQL-Server-connector.html)
 - [Oracle connector](Oracle-connector.html)
-- [DB2 Connector](DB2-connector.html)
 - [DashDB Connector](DashDB.html)
-- [DB2 iSeries Connector](DB2-iSeries-connector.html)
-- [DB2 for z/OS connector](DB2-for-z-OS.html)
+- [IBM Db2 (for Linux, Unix, Windows) Connector](DB2-connector.html)
```

Comment:
Let's use `connector` to match with the title in https://github.com/strongloop/loopback-next/pull/4409/files#diff-967acfe8350a9043a788970b5cf91223R552

Reply:
updated.